whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmDocumentType.java

```diff
@@ -0,0 +1,5 @@
+package net.ripe.db.nrtm4.persist;
+
+public enum NrtmDocumentType {
+    snapshot,
```

Comment:
Enum should be in uppercase by convention? Also you do not need a comma if there is only one type

Reply:
the RFC specifies lower case -- so instead of making the Enum upper and then adding a method just to get the same string in lower case i just defined it as lower in the first place. intellij is happy with it so i just forgot about it. there will also be "delta" as well, so we'll get back to this code and change it then.

Reply:
I think that we should use the uppercase adding a value in lower case

Reply:
Javadocs suggest constants should always be in uppercase. https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html.

Reply:
pff. what do they know about writing Java code. changed in 'transform-rpsl-to-nrtm-json'